db/migrate/20230323050938_change_properties_in_api_namespaces_from_jsonb_to_json.rb

```diff
@@ -0,0 +1,13 @@
+class ChangePropertiesInApiNamespacesFromJsonbToJson < ActiveRecord::Migration[6.1]
+  def up
+    remove_index :api_namespaces, name: "index_api_namespaces_on_properties"
+
+    change_column :api_namespaces, :properties, :json
```

Comment:
@Pralish I'm worried this is going to lock up the table for a long time in big apps, causing requests to to hang while the DB changes in the background

Can you add `strong_migrations` https://github.com/ankane/strong_migrations and see what it says?

Reply:
@donrestarone, The change is on api_namespaces table. I don't think we have a lot of api_namespaces (less than 50 per app in my opinion).

Having said that, I used the strong_migrations gem and this was the response:

```
== 20230323050938 ChangePropertiesInApiNamespacesFromJsonbToJson: migrating ===
-- remove_index(:api_namespaces, {:name=>"index_api_namespaces_on_properties"})
   -> 0.0019s
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled:

=== Dangerous operation detected #strong_migrations ===

Changing the type of an existing column blocks reads and writes
while the entire table is rewritten. A safer approach is to:

1. Create a new column
2. Write to both columns
3. Backfill data from the old column to the new column
4. Move reads from the old column to the new column
5. Stop writing to the old column
6. Drop the old column 
```

Reply:
@Pralish yeah that was my fear. Can we add strong migrations to the code base, ensure it runs in CI and then tackle this issue with their suggestion?